### ISSUE TITLE
Feature Request: Setting Presenters per Property

### ISSUE THREAD

**Issue by bendobos** [UX-and-I/storybook-design-token#11]
It would be really cool if the presenter could be overridden per property:

```
/* vars.css */
/**
 * @tokens Borders
 */
:root {
  --border-radius: 3px; /** @presenter BorderRadius */
  --border-gray: 1px solid #DDDDDD; /** @presenter Border */
}
```

That way, developers could make "mixed blocks" of properties and would not be forced to structure their CSS in a certain way to make it work with this plugin. It would add quite a bit of flexibility, in my opinion.

**Comment by angrycat9000** [comment]
Second the idea to separate the groups from the presenters. 

I tried formatting the CSS as follows but got multiple _Borders_ groups in the UI. 

```css
/** 
 * @tokens Borders
 * @presenter BorderRadius
 */
:root { --border-radius: 3px }
/**
 * @tokens Borders
 * @presenter Color
 */
:root { --border-color: #ddd }
```

**Comment by Sqrrl** [comment]
Big shout-out to @BilelJegham for implementing this. Released with v1.1.0. Example is in the Readme.
